lib/kim/Install.py

```diff
@@ -87,14 +87,15 @@ def error():
 if buildflag:
 
   # set install directory
+
   dir = os.path.join(os.path.abspath(dir), "installed-" + version)
 
-  # check to see if an installed kim-api already exists
+  # check to see if an installed kim-api already exists and wipe it out.
 
   if os.path.isdir(dir):
-     print("kim-api is already installed at %s" % dir)
-     print("Must remove this directory in order to resintall at this location")
-     sys.exit()
+    print("kim-api is already installed at %s.\nRemoving it for re-install" % dir)
```

Comment:
Either way is OK with me.  This is more of a lammps convention than anything: How should these Install.py scripts behave?...